sdk/eventhubs/azure-messaging-eventhubs/README.md

```diff
@@ -1,3 +1,189 @@
+
+Use the client library `github.com/Azure/azure-sdk-for-cpp/sdk//eventhubs` in your application to:
+
+- Send events to an event hub.
+- Consume events from an event hub.
+
+Key links:
+- [Source code][source]
+- [API Reference Documentation][godoc]
+- [Product documentation](https://azure.microsoft.com/services/event-hubs/)
+- [Samples][godoc_examples]
+
+## Getting started
+
+### Install the package
+
+Install the Azure Event Hubs client module for Go with `go get`:
+
+```bash
+vcpkg install azure-messaging-eventhubs-cpp
+```
+
+### Prerequisites
+
+- A C++ Compiler with C++14 support
+- An [Azure subscription](https://azure.microsoft.com/free/)
+- An [Event Hub namespace](https://docs.microsoft.com/azure/event-hubs/).
+- An Event Hub. You can create an event hub in your Event Hubs Namespace using the [Azure Portal](https://docs.microsoft.com/azure/event-hubs/event-hubs-create), or the [Azure CLI](https://docs.microsoft.com/azure/event-hubs/event-hubs-quickstart-cli).
+
+### Authenticate the client
+
+Event Hub clients are created using an Event Hub a credential from the [Azure Identity package][azure_identity_pkg], like [DefaultAzureCredential][default_azure_credential].
+You can also create a client using a connection string.
+
+<!-- NOTE: GO Links are intentional for now -->
+#### Using a service principal
+ - ConsumerClient: [link](https://pkg.go.dev/github.com/Azure/azure-sdk-for-go/sdk/messaging/azeventhubs#example-NewConsumerClient)
+ - ProducerClient: [link](https://pkg.go.dev/github.com/Azure/azure-sdk-for-go/sdk/messaging/azeventhubs#example-NewProducerClient)
+
+<!-- NOTE: GO Links are intentional for now -->
+#### Using a connection string
+ - ConsumerClient: [link](https://pkg.go.dev/github.com/Azure/azure-sdk-for-go/sdk/messaging/azeventhubs#example-NewConsumerClientFromConnectionString)
+ - ProducerClient: [link](https://pkg.go.dev/github.com/Azure/azure-sdk-for-go/sdk/messaging/azeventhubs#example-NewProducerClientFromConnectionString)
+
+# Key concepts
+
+An Event Hub [**namespace**](https://docs.microsoft.com/azure/event-hubs/event-hubs-features#namespace) can have multiple event hubs.
+Each event hub, in turn, contains [**partitions**](https://docs.microsoft.com/azure/event-hubs/event-hubs-features#partitions) which 
+store events.
+
+Events are published to an event hub using an [event publisher](https://docs.microsoft.com/azure/event-hubs/event-hubs-features#event-publishers). In this package, the event publisher is the [ProducerClient](https://pkg.go.dev/github.com/Azure/azure-sdk-for-go/sdk/messaging/azeventhubs#ProducerClient)
+
+Events can be consumed from an event hub using an [event consumer](https://docs.microsoft.com/azure/event-hubs/event-hubs-features#event-consumers). In this package there are two types for consuming events: 
+- The basic event consumer is the PartitionClient, in the
```

Comment:
Incomplete sentence

Reply:
Not actually incomplete - more accurately, the - on the next line was incorrect.